src/ui/components/Transcript/Transcript.css

```diff
@@ -0,0 +1,27 @@
+.sidebar {
+  background: var(--tab-bgcolor-alt-subtle);
+  display: flex;
+  flex-direction: column;
+  height: 100%;
+  width: 100%;
+}
+
+.sidebar-toolbar {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
+  border-bottom: 1px solid var(--theme-splitter-color);
+  margin-bottom: 0;
+}
+
+.sidebar-toolbar-item {
+  font-size: 15px;
+  line-height: 19px;
+  font-weight: 400;
+  color: var(--body-color);
+  overflow: hidden;
+  white-space: nowrap;
+  text-overflow: ellipsis;
+  padding: 8px;
```

Comment:
It's totally up to you, but I find that sorting in styling (when it won't change behavior) can be really nice. It tends to group together like things (in this case the two font properties) and it makes it easier to see when you might accidentally have two conflicting properties.